pkg/cli/config/config.go

```diff
@@ -70,3 +74,46 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 
 	return bpConfig, nil
 }
+
+// CheckAPIConnection validate API connection via configured proxy and VPN
+func (config BackplaneConfiguration) CheckAPIConnection() error {
+
+	// Check backplane Proxy URL
+	if config.ProxyURL == "" {
+		//Cannot connect to backplane API URL: Check if you need to use a proxy/VPN to access backplane
+		return errors.New("empty proxy url - check your backplane-cli configuration in $HOME/.config/backplane/config.json")
```

Comment:
The backplane config file could be other if it is set by env?

Reply:
Thanks @bmeng , Derive the path dynamically